addon/models/tumblr-post.js

```diff
@@ -17,5 +18,6 @@ export default DS.Model.extend({
   source_title: attr('string'),
   liked: attr('boolean'),
   state: attr('string'),
-  total_posts: attr('number')
+  total_posts: attr('number'),
+  isPhotoEntry: equal('type', 'photo')    
```

Comment:
Since we already have `type` on the model, we shouldn't add a photo-specific computed like this. It won't scale to adding more and more types. I'll add a suggestion down where you use this for a better alternative.